src/main.cpp

```diff
@@ -588,7 +588,19 @@ void setup()
     // fixed screen override?
     if (config.display.oled != meshtastic_Config_DisplayConfig_OledType_OLED_AUTO)
         screen_model = config.display.oled;
-
+#ifdef UNPHONE
+    // initialise IO expander with pinmodes
+    Wire.beginTransmission(0x26);
+    Wire.write(0x06);
+    Wire.write(0x7A);
+    Wire.write(0xDD);
+    Wire.endTransmission();
+    Wire.beginTransmission(0x26);
+    Wire.write(0x02);
+    Wire.write(0x04); // Backlight on
+    Wire.write(0x22); // G&B LEDs off
+    Wire.endTransmission();
+#endif
```

Comment:
I'd like to propose a wider scope for the use of port expanders in meshtastic. Which chip do you use exactly?
Suggestion would be love leave this as-is and repalce later with proper handling of port expanders.